src/icon2svg.rs

```diff
@@ -0,0 +1,163 @@
+//! Produces svgs of icons in Google-style icon fonts
+
+use crate::{error::DrawSvgError, iconid::IconIdentifier};
+use kurbo::{Affine, BezPath, PathEl, Point};
+use skrifa::{
+    instance::{LocationRef, Size},
+    outline::DrawSettings,
+    raw::TableProvider,
+    FontRef, MetadataProvider,
+};
+use write_fonts::pens::{BezPathPen, TransformPen};
+
+fn round2(v: f64) -> f64 {
+    (v * 100.0).round() / 100.0
+}
+
+fn push_point(svg: &mut String, prefix: char, p: Point) {
+    svg.push(prefix);
+    svg.push_str(&format!("{},{}", round2(p.x), round2(p.y)));
+}
+
+/// We use this rather than [`BezPath::to_svg`]` so we can exactly match the output of the tool we seek to replace
+fn push_drawing_commands(svg: &mut String, path: &BezPath) {
```

Comment:
this is making me wonder if a `SvgObject` trait wouldn't make sense; it would have one method that looked like `Display::fmt`, where some type could write itself as SVG into a buffer. Just an idea though, not something I would change in this PR.